### PR TITLE
E-928 feat(class): add DataEventSource and update deps

### PR DESCRIPTION
**Pull request by dudiq**

### CODE REVIEW

File: packages/class/src/DataEventSource.js [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+    this.eventSource.removeEventListener('message', this.handleEvent);
+    this.eventSource.close();
+    this.eventSource = null;
+  }
+
+  /**
+   * @private
+   * @param {MessageEvent} data
+   */
+  handleEvent = data => {
```
Comment (lamartire) [inline packages/class/src/DataEventSource.js:45]: Why arrow function?

Comment (dudiq) [reply packages/class/src/DataEventSource.js:45]: need store context when call
`this.eventSource.addEventListener('message', this.handleEvent);`
and then
`this.eventSource.removeEventListener('message', this.handleEvent);`

Comment (lamartire) [reply packages/class/src/DataEventSource.js:45]: Not obvious. Can't using "just method", without arrow?

Comment (dudiq) [reply packages/class/src/DataEventSource.js:45]: nope, can't using without arrow,
another solution is create handler in constructor with bind for pass the same handler to eventSource.
i prefer arrow solution

File: packages/class/types/Request.d.ts [hunk 2]
```diff
@@ -8,6 +8,10 @@ export type RequestCreateParams = {
   createAnswer?: RequestAnswerFactory;
 };
 
+export type UploadField = {
```
Comment (kazurus) [inline packages/class/types/Request.d.ts:11]: What's the difference with `object`?

Comment (dudiq) [reply packages/class/types/Request.d.ts:11]: after update to latest TS, `object` does not passed

File: packages/class/package.json [hunk 3]
```diff
@@ -53,10 +53,10 @@
     "url": "https://github.com/endpass/endpass-core/issues"
   },
   "dependencies": {
-    "@endpass/utils": "^1.8.3",
```
Comment (kazurus) [inline packages/class/package.json:56]: update the version for `class package`

**Pull request merged**